simulator/func_sim/alu.h

```diff
@@ -236,6 +236,13 @@ struct ALU
 
     // Bit manipulations
     template<typename I> static void sbext( I* instr) { instr->v_dst[0] = 1U & ( instr->v_src[0] >> shamt_v_src2<typename I::RegisterUInt>( instr)); }
+    
+    template<typename I> static void max( I* instr)
```

Comment:
This is not in our code guidelines documents yet, but usually we put `template<> static` to a new line (see `riscv_unshfl` above).